src/vswhere/Program.cpp

```diff
@@ -12,6 +14,10 @@ void WriteLogo(_In_ const CommandArgs& args, _In_ wostream& out);
 
 int wmain(_In_ int argc, _In_ LPCWSTR argv[])
 {
+    _setmode(_fileno(stdout), _O_U16TEXT);
```

Comment:
This looks better and I may well take it, but talking with the VC++ team if there's a better way with STL streams.